site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -424,12 +422,77 @@ When enabled the user's password hash will be modified if it does not match the
 
 
 ==== Webhooks
-image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configuration - Webhooks,width=1200,role=top-cropped]
+image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configuration - Webhooks,width=1200,role=shadowed]
 
 ===== Table Columns
 
 include::docs/v1/tech/events-webhooks/_tenant-webhooks-table.adoc[]
 
+==== Security
+
+image::core-concepts/tenant-configuration-security-acl-catpcha.png[Tenant Configuration - Security: ACL and Captcha,width=1200,role=bottom-cropped]
+
+===== Login API settings
+
+[.api]
+[field]#Require an API key# [optional]#Optional#::
+When enabled the Login API will require an API key. This is functionally equivalent to requiring client authentication during OAuth2.
+
+===== Access control lists settings
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[.api]
+[field]#Access control list# [optional]#Optional#::
+The IP access control list that will be applied to this tenant.
+
+===== Captcha settings
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[.api]
+[field]#Enabled# [optional]#Optional#::
+When enabled, Captcha is used to help increase security of a form submission on the FusionAuth themed pages.
+
+[field]#Method# [required]#Required#::
+The type of Captcha to use. FusionAuth supports `Google reCAPTCHA v2`, `Google reCAPTCHA v3`, `hCaptcha`, and `hCaptcha Enterprise`.
+
+[field]#Secret key# [required]#Required#::
+The secret key for this Captcha service.
+
+[field]#Site key# [required]#Required#::
+The site key for this Captcha service.
+
+[field]#Site key# [optional]#Optional#::
+The threat score threshold for this Captcha service if required. If it is not used by this Captcha service then the value will be ignored.
+
+image::core-concepts/tenant-configuration-security-domains-rate-limiting.png[Tenant Configuration - Security: Blocked domains and rate limiting,width=1200,role=top-cropped]
+
+===== Blocked domain settings
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[.api]
+[field]#Blocked domains# [optional]#Optional#::
+One or more line separated domains that are not allowed to register when self service is enabled. For example, you may wish to disallow your company domain to prevent users employees using the self-service registration form, or to prevent end users from attempting to create an account using a company email address.
+
+===== Rate limit settings
```

Comment:
We should describe the failed login rate limiting and how it works alongside the existing failed login configuration.